pkgs/kde/frameworks/kimageformats/default.nix

```diff
@@ -15,7 +15,8 @@ mkKdeDerivation {
   extraBuildInputs = [
     libheif
     libjxl
-    libavif
+    # FIXME: cmake files are broken, disabled for now
+    # libavif
```

Comment:
Upstream?

Reply:
No, this is entirely our problem, #417080.

Reply:
Would it be better to just revert then?

Reply:
50k rebuilds says no.

Reply:
Oh I missed that this is `extraBuildInputs`. Should we revert on staging?